difference-of-squares/difference_of_squares.rb

```diff
@@ -0,0 +1,36 @@
+class Squares
+  def initialize(num)
+    number=num
```

Comment:
@kalgibhatt-btc Solve spaces like `number=num` for each and every line of code and remember it for future. It should look like `number = num`.